Makefile

```diff
@@ -47,6 +47,22 @@ test: test-go test-js
 shorttest:
 	go test -mod vendor -p $(GO_PARALLEL_JOBS) -short -tags skipcontainertests,skiplargetiltfiletests -timeout 100s ./...
 
+# Run recent changes as seen by Tilt, called from Tiltfile
+uniq = $(if $1,$(firstword $1) $(call uniq,$(filter-out $(firstword $1),$1)))
```

Comment:
would it make more sense to do this in a shell script?

i feel like i've gotten in trouble with stuff that slows down evaluation of the makefile -- and there doesn't seem to be a strong reason to do it in the makefile.

this is fine too though

Reply:
Compromise in 09ff0bb6f715bf9be53f9fbafd88e4095fe7f255 that only makes the target visible if run from inside Tilt/Tiltfile so that regular make commands are unaffected by the extra processing?